main.py

```diff
@@ -17,6 +17,8 @@ def main():
 
     console.print_greeting()
     console.print_instructions()
+    board_size = console.select_board_size(
+        "Let's build a board! Pick an odd number from 3 to 5: ", validator)
```

Comment:
I like the intention here to show what range is available; the wording is a bit confusing because there is no odd number between 3 and 5.

Reply:
I agree so much with this. I'll work on the wording of this.